packages/jmix-react-ui/src/ui/table/DataTable.tsx

```diff
@@ -504,7 +504,7 @@ class DataTableComponent<
           {this.props.buttons}
           {this.props.hideClearFilters ? null : this.clearFiltersButton}
         </div>
-        <Table { ...tableProps } className={this.props.hideSelectionColumn ? '_cuba-hide-selection-column' : ''} />
+        <Table { ...tableProps } />
```

Comment:
Please also remove `hideSelectionColumn` prop and selector with `_cuba-hide-selection-column`.

Reply:
I left hideSelectionColumn and just added a condition to render nothing if hideSelectionColumn is provided. 

https://github.com/Haulmont/jmix-frontend/pull/456/commits/66181e67be6bf3518cdd68acd43d74be47376328